### DISCUSSION TITLE
ThinkPad Z13 SSD replacement - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#29] [General]
# ThinkPad Z13 SSD replacement - Alan Pope's blog

One thing I’ve loved about ThinkPad laptops for many years, is the upgradability and repairability. Ever since the early days, it’s been possible to noodle around inside a ThinkPad.
Sadly, some of the modern X1/Ultrabook line, and some others, are less upgradable than previous generations. My year-old ThinkPad Z13 has a few options for noodling around inside though, including storage.
My Z13 shipped with Ubuntu out of the box. I wanted to try something different, but didn’t want to wipe the OS off the SSD.

https://popey.com/blog/2023/09/thinkpad-z13-ssd-replacement/

**Comment by beardChamp** [comment]
It might be worth checking out wether you have a WWLAN slot and if it can be used as a secondary drive. I've got a T480 that allows that. I've been trying out a few new distros and it's kind of nice to have a secondary drive that I can throw the ISOs onto.
